packages/headless/src/features/commerce/facets/facet-set/facet-set-slice.ts

```diff
@@ -122,6 +128,27 @@ export const commerceFacetSetReducer = createReducer(
         updateExistingFacetValueState(existingValue, 'select');
         facetRequest.freezeCurrentValues = true;
       })
+      .addCase(toggleSelectLocationFacetValue, (state, action) => {
+        const {facetId, selection} = action.payload;
+        const facetRequest = state[facetId]?.request;
+
+        if (!facetRequest || !ensureLocationFacetRequest(facetRequest)) {
+          return;
+        }
+
+        facetRequest.preventAutoSelect = true;
```

Comment:
There is no preventAutoSelect on location facets. You can safely remove this.

Reply:
Will tackle this in https://github.com/coveo/ui-kit/pull/4574